include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -80,6 +85,131 @@ DEAL_II_NAMESPACE_OPEN
 
 namespace VectorTools
 {
+  namespace implementation
```

Comment:
To explain, this order has the effect of putting the documentation for these functions onto a separate doxygen page. There is no point in the doxygen page for `namespace VectorTools` to contain a link to `namespace VectorTools::internal` -- it's of no interest to users.